.config/guardian/.gdnbaselines

```diff
@@ -24,7 +24,7 @@
       "tool": "psscriptanalyzer",
       "ruleId": "PSAvoidUsingUsernameAndPasswordParams",

```

Comment:
> We do not have control over external components.

We do have control over `eng/common/SetupNugetSources.ps1` that's from arcade.  Maybe we should delete this one?